package.json

```diff
@@ -92,6 +93,7 @@
     "jest-watch-typeahead": "0.4.2",
     "lage": "0.27.0",
     "lerna": "^3.15.0",
+    "lerna-dependency-graph": "^1.0.2",
```

Comment:
Should be this under `scripts/package.json`?

Reply:
oh whoops, yeah this should definitely by in `scripts`

Reply:
I added an arg for this, is that good enough ? or should I add a default windows path

Reply:
moved